server/src/test/java/org/elasticsearch/gateway/ReplicaShardAllocatorIT.java

```diff
@@ -133,7 +136,7 @@ public void testRecentPrimaryInformation() throws Exception {
             // AllocationService only calls GatewayAllocator if there are unassigned shards
             execute("""
                 create table doc.dummy (x int)
-                with ("routing.allocation.require.attr" = 'not-found')
+                with ("number_of_replicas" = 1)
```

Comment:
shouldn't this also contain `wait_for_active_shards=0`? at least the original test does this..

Reply:
But looks like this has no effect at this test and such is unrelated, so ignore my comment.

Reply:
I'll add it anyway :+1: